src/pages/resources/custom-asset-compute-worker/lesson3.md

```diff
@@ -45,29 +45,34 @@ IMGIX_SECURE_TOKEN=imgx-token
 IMGIX_DOMAIN=your-subdomain.imgix.net
 ```
 
-Edit `manifest.yml` file and add `inputs` object, as shown below. This file describes IO Runtime action to be deployed.
+Edit `app.config.yaml` file and add `inputs` object, as shown below. This file describes IO Runtime action to be deployed.
```

Comment:
For asset compute, the manifest info now moves to ext.config.yaml inside the extension folder, not app.